modules/solid_mechanics/include/materials/PlaneStrain.h

```diff
@@ -8,11 +8,14 @@
 #define PLANESTRAIN_H
 
 #include "Element.h"
+#include "ScalarCoupleable.h"
 
 namespace SolidMechanics
 {
 
-class PlaneStrain : public Element
+class PlaneStrain :
+  public Element,
+  ScalarCoupleable
```

Comment:
I didn't actually intend to privately inherit ScalarCoupleable -- it sounds like you have to list public before each class you inherit from for them to be public.  It probably doesn't make much difference, but I'll change it to be public.